extensions/attention-stream/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "rally-attention-stream",
   "version": "0.4.1",
   "scripts": {
-    "build": "cp manifest.prod.json manifest.json && npm run build:glean && rollup -c && npm run tailwind",
+    "build": "cp manifest.prod.json manifest.json && npm run build:glean && rollup -c 2>&1 && npm run tailwind 2>&1",
```

Comment:
we had the `2>&1` hack for jest since it seemed like it wrote to stderr as part of normal operation, but it looks like for rollup, tailwind, and web-ext, we're seeing legitimate warnings, right? I know we want to get moving on this but I'm wary of letting the warnings stack up.

If you do want to move forward though, I'd suggest changing `allowWarningsInSuccessfulBuilds` to be true, rather than redirecting output here. I think we should reserve redirecting output only for cases like jest where stderr output isn't necessarily a cause for concern.

Reply:
(flipping the flag means we can easily flip it back later and look at which builds + tests are producing noteworthy warnings)

Reply:
OK here's the output:

```
src/background.ts → dist/background.js...
(!) `this` has been rewritten to `undefined`
https://rollupjs.org/guide/en/#error-this-is-undefined
../../common/temp/node_modules/@mozilla/glean/dist/core/pings/ping_type.js
1: var __classPrivateFieldSet = (this && this.__classPrivateFieldSet) || function (receiver, state, value, kind, f) {
                                 ^
2:     if (kind === "m") throw new TypeError("Private method is not writable");
3:     if (kind === "a" && !f) throw new TypeError("Private accessor was defined without a setter");
...and 3 other occurrences
  ...28 lines omitted...
                               ~~~~~~~~~~~~~~~~~~~~~~

```

I believe this is something the Glean team is aware of, and we might not be able to do much about it without changes there ^

```
(!) Plugin typescript: @rollup/plugin-typescript: Rollup 'sourcemap' option must be set to generate source maps.
created dist/main.content.js in 4.2s
warn - The `purge`/`content` options have changed in Tailwind CSS v3.0.
warn - Update your configuration file to eliminate this warning.
warn - https://tailwindcss.com/docs/upgrade-guide#configure-content-sources
```

This is the tailwind warning, I can easily get rid of that. Would be nice if there was a way to allow warnings matching a certain pattern only etc., I wonder if rush supports that.

Reply:
OK, I think due to the glean error we're probably better off doing the `2>&1` trick just for rollup, wdyt?

Reply:
OK I would rather treat any warnings as errors that we can, it looks like glean and tailwind are the problems here.

Reply:
yes this makes sense! tracking lingering ones with issues sounds like a good way to go.